docs/contrib.rst

```diff
@@ -0,0 +1,16 @@
+
+Contributing
+------------
+
+``datatable`` is an open source project released under the Mozilla Public Licence v2. Open Source projects live by their user and developer communities. We welcome and encourage your contributions of any kind!
```

Comment:
So `datatable` is still MPL2, while FTRL (at least this is how we put it on the conference slides) can be considered as MIT?

Reply:
Yes, whatever that means. Since MPL is not a repository-level license, it only applies to those files that were explicitly marked as MPL.
We have a task #1310 to switch entirely to MIT, but I can't proceed with that without @arnocandel 's permission.